Antidote/DialingCallViewController.m

```diff
@@ -39,6 +39,28 @@ - (void)viewDidLoad
     [self installConstraints];
 }
 
+- (void)viewDidLayoutSubviews
+{
+    [self updateFriendAvatar];
+}
```

Comment:
I removed it form `viewDidLayoutSubviews`. The only reason we would call `updateFriendAvatar` is if the name of the person changes or we are adding in a name. In this case I would like to recreate the image.